AMW_web/pom.xml

```diff
@@ -82,11 +82,10 @@
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-lang3</artifactId>
 		</dependency>
-		<dependency>
-			<groupId>net.sf.json-lib</groupId>
-			<artifactId>json-lib</artifactId>
-			<classifier>jdk15</classifier>
-		</dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Please fix indent and add `<scope>provided</scope>`

Reply:
* indentation fixed (was not visible in my IDE, since this specific pom.xml uses tabs)
* `<scope>provided</scope>` is already inherited from `dependencyManagement`. added it anyway to avoid misunderstandings